lib/kitchen/directions/bake_book_answer_key/eob.xhtml.erb

```diff
@@ -0,0 +1,9 @@
+<div class="os-eob os-solutions-container" data-type="composite-chapter" data-uuid-key=".solutions">
+  <h1 data-type="document-title" id="composite-chapter-1">
+    <span class="os-text"><%= I18n.t(:eoc_answer_key_title) %></span>
+  </h1>
+  <div data-type="metadata" style="display: none;">
+    <h1 data-type="document-title" itemprop="name"><%= I18n.t(:eoc_answer_key_title) %></h1>
+    <%= @metadata.paste %>
+  </div>
+</div>
```

Comment:
The name `eob` refers to many different things, so I think this name is going to get confusing down the line. Mind renaming this template to `solutions-container` or `answer-key-wrapper` or something like that?